packages/app/src/specs/SpecsListRowItem.vue

```diff
@@ -1,14 +1,40 @@
 <template>
-  <div
-    class="h-full outline-none border-gray-50 ring-inset grid grid-cols-2 group focus-within:ring-indigo-300 focus-within:ring-1 children:cursor-pointer"
-    data-cy="specs-list-row"
-  >
-    <div>
-      <slot name="file" />
-    </div>
-
-    <div>
-      <slot name="git-info" />
-    </div>
+  <div data-cy="specs-list-row">
+    <component
+      :is="isLeaf ? 'RouterLink' : 'div'"
+      class="h-full outline-none border-gray-50 ring-inset grid group focus:outline-transparent focus-within:ring-indigo-300 focus-within:ring-1 children:cursor-pointer"
```

Comment:
For the directory rows, it looks like there's some conflict between the transparent outline focus styles and the top border that acts as the divider:

![Screen Shot 2022-06-06 at 4 52 35 PM](https://user-images.githubusercontent.com/1711637/172255503-969a63a1-8958-4f38-8b56-fbd032f4592f.png)

You might just need to use `focus:outline-none` within the RowDirectory component and let these focus-within styles take precedent.